modules/raft/src/integrationTest/java/org/apache/ignite/internal/raft/ItLearnersTest.java

```diff
@@ -409,19 +387,17 @@ void testChangePeersToAddLearnerToSameNodeAsPeer() throws InterruptedException {
 
         PeersAndLearners newConfiguration = createConfiguration(followers, List.of(learner, newLearner));
 
-        CompletableFuture<Void> changePeersFuture = learnerService.thenCompose(s -> s.refreshAndGetLeaderWithTerm()
-                .thenCompose(leaderWithTerm -> s.changePeersAndLearnersAsync(newConfiguration, leaderWithTerm.term())
-        ));
+        CompletableFuture<Void> changePeersFuture = learnerService.refreshAndGetLeaderWithTerm()
+                .thenCompose(leaderWithTerm -> learnerService.changePeersAndLearnersAsync(newConfiguration, leaderWithTerm.term()));
 
         assertThat(changePeersFuture, willCompleteSuccessfully());
 
         var newLearnerListener = new TestRaftGroupListener();
 
-        CompletableFuture<RaftGroupService> newLearnerService = startRaftGroup(
+        startRaftGroup(
                 newLearner, newConfiguration.learner(newLearner.consistentId()), newConfiguration, newLearnerListener
```

Comment:
Same as above. One at a line. Up to you though.

Reply:
Agree